GAMES_LIST.md

```diff
@@ -180,12 +191,16 @@
 | `nucleardawn`                         | Nuclear Dawn (2011)                                     | [Valve Protocol](#valve)                    |
 | `ohd`                                 | Operation: Harsh Doorstop (2023)                        | [Valve Protocol](#valve)                    |
 | `onset`                               | Onset (2019)                                            | [Valve Protocol](#valve)                    |
+| `ohd`                                 | Operation: Harsh Doorstop (2023)                        | [Valve Protocol](#valve)                    |
 | `openarena`                           | OpenArena (2005)                                        |                                             |
 | `openttd`                             | OpenTTD (2004)                                          |                                             |
 | `operationflashpoint`<br>`flashpoint` | Operation Flashpoint: Cold War Crisis (2001)            |                                             |
 | `flashpointresistance`                | Operation Flashpoint: Resistance (2002)                 |                                             |
 | `painkiller`                          | Painkiller                                              |                                             |
+| `pc`                                  | Project Cars (2015)                                     | [Valve Protocol](#valve)                    |
+| `pc2`                                 | Project Cars 2 (2017)                                   | [Valve Protocol](#valve)                    |
```

Comment:
Regarding the ID naming convention, lets dont make 2-characters ids anymore, meaning that `pc` here will become `projectcars`, and since `pc2` is a 3-characters ID, its not really intuitive so it will become `projectcars2`.